cloudformation/jolly-roger.yaml

```diff
@@ -798,9 +803,11 @@ Resources:
 
                   ${PapertrailDockerConfig}
 
-                  - docker run --name jolly-roger -d --network=host --restart=unless-stopped -e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION -e PORT=3000 -e ROOT_URL=https://${AppUrl} ghcr.io/deathandmayhem/jolly-roger
+                  - docker run --name coturn -d --restart=unless-stopped --network=host instrumentisto/coturn --min-port=50000 --log-file=stdout --external-ip='$(detect-external-ip)' --realm=${DirectUrl} --use-auth-secret --static-auth-secret=${TurnSecret}
```

Comment:
Oh it looks like the coturn/coturn docker image will do external IP detection via [DNS to the same servers we use in `imports/server/mediasoup.ts`](https://github.com/coturn/coturn/blob/master/docker/coturn/rootfs/usr/local/bin/detect-external-ip.sh) if you set `DETECT_EXTERNAL_IP` per https://github.com/coturn/coturn/blob/master/docker/coturn/README.md#automatic-detection-of-external-ip

Reply:
Haha. So I remember we had this discussion, and I _thought_ that the instrumentisto image was the one we switched _to_ (because it was the most recent image in the git history). But I think what actually happened is that we talked about it, but never actually switched, because we just got rid of coturn entirely.

(The reason that we're using those DNS servers is because we stole them from that coturn container)

I'll swap it over.